ouroboros-consensus/test-consensus/Test/Dynamic/Util.hs

```diff
@@ -261,92 +239,14 @@ leaderScheduleFromTrace (NumSlots numSlots) = LeaderSchedule .
         | nid `elem` xs = xs
         | otherwise     = nid : xs
 
-{-------------------------------------------------------------------------------
-  Crowded Run - longest multi-leader section of a leader schedule
--------------------------------------------------------------------------------}
-
--- | Describes a sequence of slots in a leader schedule with slots with
--- more than one leader, possibly interrupted by slots without leader.
--- There can be no such sequence, but if there is, first slot and number of
--- multi-leader slots are given.
-newtype CrowdedRun = CrowdedRun (Maybe (SlotNo, Word64))
-    deriving (Show, Eq)
-
-crowdedRunLength :: CrowdedRun -> Word64
-crowdedRunLength (CrowdedRun m) = maybe 0 snd m
-
-instance Ord CrowdedRun where
-    compare = compare `on` crowdedRunLength
-
-noRun :: CrowdedRun
-noRun = CrowdedRun Nothing
-
-incCrowdedRun :: SlotNo -> CrowdedRun -> CrowdedRun
-incCrowdedRun slot (CrowdedRun Nothing)          = CrowdedRun (Just (slot, 1))
-incCrowdedRun _    (CrowdedRun (Just (slot, n))) = CrowdedRun (Just (slot, n + 1))
-
--- | The longest 'CrowdedRun' of a 'LeaderSchedule'. The tiebreaker favors
```

Comment:
Hmm, `determineForkLength` is pretty simple and self-explanatory for me (and you, of course :slightly_smiling_face:), but is that true for people that have never read this explanation? Maybe you can copy an abridged version of this explanation to `determineForkLength`?

Reply:
Done